fossunited/ticketing/doctype/foss_event_ticket/test_foss_event_ticket.py

```diff
@@ -78,3 +88,59 @@ def test_checkin_as_non_chapter_member(self):
         # Then the user should not have permission to check-in the attendee
         with self.assertRaises(frappe.PermissionError):
             checkin_attendee(self.event.name, ticket.as_dict(), "test2@example.com")
+
+    def test_create_when_tickets_closed(self):
+        # Given a paid event with ticket status as "Closed"
+        self.event.tickets_status = "Closed"
+        self.event.save()
+        # When a ticket is created for this event
+        # Then an error must be thrown
+        with self.assertRaises(frappe.PermissionError):
+            insert_test_ticket(event=self.event.name)
+
+    def test_ticket_creation(self):
+        # Given an event
+        # When the tickets_status is Live
+        self.event.tickets_status = "Live"
+        self.event.save()
+
+        # Then a ticket for this event should be created without any problem
+        ticket = insert_test_ticket(event=self.event.name)
```

Comment:
Maybe add an assertion on 1-2 ticket details?

Reply:
I think testing that on Razorpay Payment capture would make sense, adding multiple ticket creation test there